diffpy/pdfmorph/pdfmorphapp.py

```diff
@@ -245,8 +245,8 @@ def main():
         if opts.savefile == "-":
             outfile = sys.stdout
         else:
-            outfile = file(opts.savefile, 'w')
-        print >> outfile, header
+            outfile = open(opts.savefile, 'w')
+        print(header, file=outfile)
         import numpy
```

Comment:
Should this be at the top?

Reply:
@pavoljuhas any insight?

Reply:
Yes, I would move `import numpy` to the  top.

@chiahaoliu - please apply this patch as numpy.savetxt does not work with zip iterator
```diff
diff --git a/diffpy/pdfmorph/pdfmorphapp.py b/diffpy/pdfmorph/pdfmorphapp.py
index 851ddd9..a692453 100755
--- a/diffpy/pdfmorph/pdfmorphapp.py
+++ b/diffpy/pdfmorph/pdfmorphapp.py
@@ -248,7 +248,8 @@ def main():
             outfile = open(opts.savefile, 'w')
         print(header, file=outfile)
         import numpy
-        numpy.savetxt(outfile, zip(chain.xobjout, chain.yobjout))
+        numpy.savetxt(outfile,
+                      numpy.transpose([chain.xobjout, chain.yobjout]))
         outfile.close()
 
     if opts.plot:
```